addons/sale/tests/test_credit_limit.py

```diff
@@ -0,0 +1,46 @@
+from odoo.addons.sale.tests.common import SaleCommon
+from odoo.fields import Command
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestSaleOrderCreditLimit(SaleCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.env.company.account_use_credit_limit = True
+
+        buck_currency = cls.env['res.currency'].create({
+            'name': 'Test Buck ',
+            'symbol': 'TB',
+        })
+        cls.env['res.currency.rate'].create({
+            'name': '2023-01-01',
+            'rate': 2.0,
+            'currency_id': buck_currency.id,
+            'company_id': cls.env.company.id,
+        })
+
+        cls.buck_pricelist = cls.env['product.pricelist'].create({
```

Comment:
You still have the following rate: '1 USD = 1.308130 Tes'... It would have been better to explain in your test why.

Reply:
You're right, nice catch :+1: 
I thought creating a dedicated rate would fix it, but it didn't, my bad :blush: 
The problem comes from the fact that we are still using the demo company, with all its demo data (not good indeed).
The easiest solution, I think, is to create a new/clean company.